app/src/hooks/useCheckoutContext/useCheckoutContext.tsx

```diff
@@ -0,0 +1,13 @@
+import { useContext } from "react";
+
+import { CheckoutContext } from "context/checkout/CheckoutContext";
+
+export const useCheckoutContext = () => {
+  const context = useContext(CheckoutContext);
+
+  if (context === undefined) {
```

Comment:
Why not `if (!context) ` ?

Reply:
Could be also. We are certain that the return type is `undefined` so both work. I'll do it in the next context hooks 👍